File: Ejercicio5_Java_Servlets/src/main/java/datos/Usuarios.java
package datos;

import java.util.ArrayList;
import java.util.List;

public class Usuarios {
	
	private List<Usuario> usuarios;

	public Usuarios() {
		Usuario u1 = new Usuario();
		Usuario u2 = new Usuario();
		Usuario u3 = new Usuario();
		
		u1.setUsername("username1");
		u1.setPassword("passwd1");
		
		u2.setUsername("username2");
		u2.setPassword("passwd2");
		
		u3.setUsername("username3");
		u3.setPassword("passwd2");
		
		usuarios = new ArrayList();
		
		usuarios.add(u1);
		usuarios.add(u2);
		usuarios.add(u3);
	}

	public List<Usuario> getUsuarios() {
		return usuarios;
	}

	public void setUsuarios(List<Usuario> usuarios) {
		this.usuarios = usuarios;
	}	
}

File: Ejercicio12_Angular_VJ2/src/app/datos/Usuarios.ts
import { Usuario } from '../datos/Usuario';

export class Usuarios {
	
	private _usuarios: Array<Usuario>;

	constructor() {
		this._usuarios = [];
		
		let u1: Usuario = new Usuario("username1", "passwd1");
		this._usuarios.push(u1);

		u1 = new Usuario("username2", "passwd2");
		this._usuarios.push(u1);

		u1 = new Usuario("username3", "passwd3");
		this._usuarios.push(u1);
	}

	public get usuarios(): Array<Usuario> {
        return this._usuarios;
    }

    public set usuarios(usuarios: Array<Usuario>) {
        this._usuarios = usuarios;
    }	
}
